scripts/build-python-client.sh

```diff
@@ -30,4 +30,8 @@ sed -i 's/from github.com/from armada_client.github.com/g' client/python/armada_
 sed -i 's/from google.api/from armada_client.google.api/g' client/python/armada_client/armada/*.py
 sed -i 's/from google.api/from armada_client.google.api/g' client/python/armada_client/google/api/*.py
 
-find client/python/armada_client/ -name '*.py' | xargs sed -i 's/from k8s.io/from armada_client.k8s.io/g'
\ No newline at end of file
+find client/python/armada_client/ -name '*.py' | xargs sed -i 's/from k8s.io/from armada_client.k8s.io/g'
+
+# Generate better docs for the client
+export PYTHONPATH=${PWD}/client/python
+python3.8 ${PWD}/client/python/armada_client/gen/event_typings.py
```

Comment:
```suggestion
python3 ${PWD}/client/python/armada_client/gen/event_typings.py
```

Minor suggestion.  If we update the docker file, it would be nice to not have to update this file.  I think python3 should work instead.